packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -370,7 +368,7 @@ class _TreemapState extends State<Treemap> {
           return const SizedBox();
         }
         final positionedChildren = buildTreemaps(
-          children: widget.nodes,
+          children: widget.nodes!,
```

Comment:
add an assert at the beginning of this method: `assert(widget.nodes?.isNotEmpty ?? false)`

Reply:
`widget.nodes != null && widget.nodes!.isNotEmpty` seems to be more readable